docs/contributing.rst

```diff
@@ -248,6 +248,7 @@ All integrations live in ``ddtrace/contrib/`` and contain at least two files,
 
 It is preferred to keep as much code as possible in ``patch.py``.
 
+All spans generated by the integration must add the tag ``component:<integration_name>`` to each span.
```

Comment:
Can we add an example (or link to one).

@wconti27 @brettlangdon Are there any other tags that should be set in every integration that we should document here? (ex: span.kind, _dd.measured, _dd.tracer_kr, etc.)

Reply:
sure, sounds good

Reply:
let's add them as we make them consistent, wdyt @wconti27? (asking that you could own updating this doc with any new PRs you have)